code/__DEFINES/mobs.dm

```diff
@@ -56,6 +56,7 @@
 //Organ defines for carbon mobs
 #define ORGAN_ORGANIC 1
 #define ORGAN_ROBOTIC 2
+#define ORGAN_MINERAL 3 // Used for the plasmaman liver
```

Comment:
I would refactor all the organ flags to use mob_biotype flags for this.  `MOB_MINERAL`, `MOB_HUMANOID`, etc.  Might be way out of scope for the PR though.

Reply:
I agree, and that's something I was wanting to do but it would make this PR even more annoying to review so I withheld for now. I'd have to touch every single piece of code that checks for robotic organs. It is a step in the right direction at least.